libs/utils/utils.js

```diff
@@ -1163,8 +1173,10 @@ async function processSection(section, config, isDoc) {
   }
 
   if (section.preloadLinks.length) {
-    const preloads = section.preloadLinks.map((block) => loadBlock(block));
+    const [modals, nonModals] = partition(section.preloadLinks, (block) => block.classList.contains('modal'));
+    const preloads = nonModals.map((block) => loadBlock(block));
```

Comment:
If we don't ever need to wait for modals, should we just filter them before they even land in the preload section and let them load normally later on?
So in my mind the fix is filtering modals completely from here: https://github.com/adobecom/milo/blob/stage/libs/utils/utils.js#L798